packages/api/src/lib/granule-delete.ts

```diff
@@ -26,7 +26,7 @@ const logger = new Logger({ sender: '@cumulus/api/granule-delete' });
  * @returns {Promise<void>}
  */
 const _deleteS3Files = async (
-  files: (ApiFile | PostgresFileRecord)[] = []
+  files: (Omit<ApiFile, 'granuleId'> | PostgresFileRecord)[] = []
```

Comment:
This was done to match `ApiGranule` where we have:

`files?: Omit<import('./files').ApiFile, 'granuleId'>[]`

If we use this delete function on a file that came from an ApiGranule, we'll need to drop `granuleId` from the expected type.

Reply:
I don't love it, but I do love keeping granule files and the file object coupled.